src/odemis/driver/orsay.py

```diff
@@ -1389,3 +1440,1247 @@ def terminate(self):
+        self._steererStigmatorConnector = None
+        self.steererShift = model.TupleContinuous((0.0, 0.0), unit="V", range=[(-100.0, -100.0), (100.0, 100.0)])
+        self._steererShiftConnector = None
+        self.steererTilt = model.TupleContinuous((0.0, 0.0), unit="V", range=[(-10.0, -10.0), (10.0, 10.0)])
+        self._steererTiltConnector = None
+        self.orthogonality = model.FloatContinuous(0.0, unit="rad", range=(-pi, pi))
+        self._orthogonalityConnector = None
+        self.objectiveRotationOffset = model.FloatContinuous(0.0, unit="rad", range=(0, 2 * pi))
+        self._objectiveRotationOffsetConnector = None
+        self.objectiveStageRotationOffset = model.FloatContinuous(0.0, unit="rad", range=(-pi, pi))
+        self._objectiveStageRotationOffsetConnector = None
+        self.tilt = model.TupleContinuous((0.0, 0.0), unit="rad", range=[(-pi, -pi), (pi, pi)])
+        self._tiltConnector = None
+        self.xyRatio = model.FloatContinuous(1.0, unit="rad", range=(0.0, 2.0))
+        self._xyRatioConnector = None
+        self.mirror = model.BooleanVA(False)
+        self._mirrorConnector = None
+        self.imageFromSteerers = model.BooleanVA(False)
+        self._imageFromSteerersConnector = None
+        self.objectiveVoltage = model.FloatContinuous(0.0, unit="V", range=(0.0, 2e4))
+        self._objectiveVoltageConnector = None
+        self.beamShift = model.TupleContinuous((0.0, 0.0), unit="m", range=[(-1.0e-4, -1.0e-4), (1.0e-4, 1.0e-4)])
+        self._beamShiftConnector = None
+        self.horizontalFOV = model.FloatContinuous(0.0, unit="m", range=(0.0, 1.0))
+        self._horizontalFOVConnector = None
+        self.measuringCurrent = model.BooleanVA(False)
+        self._measuringCurrentConnector = None
+        self.current = model.FloatContinuous(0.0, readonly=True, unit="A", range=(0.0, 1.0e-5))
+        self._currentConnector = None
+        self.videoDelay = model.FloatContinuous(0.0, unit="s", range=(0, 1e-3))
+        self._videoDelayConnector = None
+        self.flybackTime = model.FloatContinuous(0.0, unit="s", range=(0, 1e-3))
+        self._flybackTimeConnector = None
+        self.blankingDelay = model.FloatContinuous(0.0, unit="s", range=(0, 1e-3))
+        self._blankingDelayConnector = None
+        self.rotation = model.FloatContinuous(0.0, unit="rad", range=(-pi, pi))
+        self._rotationConnector = None
+        self.dwellTime = model.FloatEnumerated(1e-7, unit="s",
+                                               choices={1e-3, 5e-4, 1e-4, 5e-5, 1e-5, 5e-6, 1e-6, 5e-7, 2e-7, 1e-7})
+        self._dwellTimeConnector = None
+        self.contrast = model.FloatContinuous(1.0, unit="", range=(0, 1))
+        self._contrastConnector = None
+        self.brightness = model.FloatContinuous(1.0, unit="", range=(0, 1))
+        self._brightnessConnector = None
+        self.operatingMode = model.BooleanVA(False)
```

Comment:
What is that? It feels strange that the "mode" can be True of False. Is that the "power"?